diff --git a/actions/adminActions.php b/actions/adminActions.php
index d192782..7c5682e 100644
--- a/actions/adminActions.php
+++ b/actions/adminActions.php
@@ -627,7 +627,7 @@ function getUsers()
                 echo '<input name="deleteUser" type="submit" class="btn btn-xs btn-link" onclick="deleteUser(' . $row[0] . ')" value="Delete" disabled />';
               }

-        if ($row[4] == '2')
+        if ($row[5] == '2')
         {
           if ( ( MODERATOR_REACTIVATE_USER == true && $_SESSION['admin_privilege'] == 1 ) || ( $_SESSION['admin_privilege'] == 2 ) )
           {
@@ -659,7 +659,7 @@ function getUsers()
                 <button name="editUser" type="button" data-toggle="modal" id="' . $row[0] . '" data-target="#editUserModal" class="btn btn-xs btn-link" disabled >Edit</button>
                 <input name="deleteUser" type="submit" class="btn btn-xs btn-link" onclick="deleteUser(' . $row[0] . ')" value="Delete" disabled />
                 ';
-        if ($row[4] == '2')
+        if ($row[5] == '2')
         {
             echo '<input name="reactivateUser" type="submit" class="btn btn-xs btn-link" value="Reactivate"  disabled/>';
         }
--
2.18.0
