diff --git a/oc-admin/ncicAdmin.php b/oc-admin/ncicAdmin.php
index 9d6818c..0f68df8 100644
--- a/oc-admin/ncicAdmin.php
+++ b/oc-admin/ncicAdmin.php
@@ -35,6 +35,13 @@ This program comes with ABSOLUTELY NO WARRANTY; Use at your own risk.
       $name = $_SESSION['name'];
     }
 
+    require_once(__DIR__ . '/../oc-config.php');
+    require_once(__DIR__ . '/../oc-functions.php');
+
+    include(__DIR__ . '/../actions/adminActions.php');
+    include(__DIR__ . '/../actions/ncicAdminActions.php');
+
+

     if ( $_SESSION['admin_privilege'] == 2)
     {
@@ -43,24 +50,18 @@ This program comes with ABSOLUTELY NO WARRANTY; Use at your own risk.
           //Do nothing
       }
     }
-    else if ($_SESSION['admin_privilege'] == 1)
+    else if ( $_SESSION['admin_privilege'] == 1 && MODERATOR_NCIC_EDITOR == true )
     {
       if ($_SESSION['admin_privilege'] == 'Moderator')
       {
-          // Do Nothing
+          //Do nothing
       }
     }
     else
     {
       die("You do not have permission to be here. This has been recorded");
-
     }
 
-    require_once(__DIR__ . '/../oc-config.php');
-    require_once(__DIR__ . '/../oc-functions.php');
-
-    include(__DIR__ . '/../actions/adminActions.php');
-    include(__DIR__ . '/../actions/ncicAdminActions.php');

     $citationMessage = "";
     if(isset($_SESSION['citationMessage']))
diff --git a/oc-admin/oc-admin-includes/sidebarNav.inc.php b/oc-admin/oc-admin-includes/sidebarNav.inc.php
index cb8d733..9fd87a4 100644
--- a/oc-admin/oc-admin-includes/sidebarNav.inc.php
+++ b/oc-admin/oc-admin-includes/sidebarNav.inc.php
@@ -5,11 +5,11 @@
     <ul class="nav side-menu">
         <li <?php if ( $pageName == "Dashboard") echo $currentPage; ?>><a href="<?php echo BASE_URL; ?>/oc-admin/admin.php"><i class="fas fa-home"></i>  Dashboard</a></li>
         <li <?php if ( $pageName == "User Management") echo $currentPage; ?>><a href="<?php echo BASE_URL; ?>/oc-admin/userManagement.php"><i class="fas fa-user"></i> User Management</a></li>
-      </li>
-      <li><a><i class="fas fa-database"></i> NCIC Editor <span class="fas fa-chevron-down"></span></a>
-        <ul class="nav child_menu">
-        </ul>
-      </li>
+        <?php if ( ( MODERATOR_NCIC_EDITOR == true && $_SESSION['admin_privilege'] == 1 ) || ( $_SESSION['admin_privilege'] == 2 ) )
+        { ?>
+        <li <?php if ( $pageName == "NCIC Editor") echo $currentPage; ?>><a href="<?php echo BASE_URL; ?>/oc-admin/ncicAdmin.php"><i class="fas fa-database"></i> NCIC Editor</a></li>
+      <?php  } else { }?>
+
         <li <?php if ( $pageName == "About OpenCAD") echo $currentPage; ?>><a href="<?php echo BASE_URL; ?>/oc-admin/about.php"><i class="fas fa-info-circle"></i> About OpenCAD</a></li>
       </li>
     </ul>
diff --git a/oc-config.sample.php b/oc-config.sample.php
index f65b86e..a7dc684 100644
--- a/oc-config.sample.php
+++ b/oc-config.sample.php
@@ -442,6 +442,18 @@ define('MODERATOR_REMOVE_GROUP', true);
   **/
 define('MODERATOR_DELETE_USER', true);

+/**#@+
+  * Moderator Settings - NCIC Editor
+  *
+  * If 'true' then Moderators will be able to access the NCIC editor,
+  * if 'false' Moderators will not be able to access the NCCIC editor.
+  *
+  * These setting will likely be moved to an *_options table in a future version.
+  *
+  * @since OpenCAD version 0.2.3
+  **/
+define('MODERATOR_NCIC_EDITOR', true);
+
 /**#@+
   * Extra Settings
 	*
--
2.17.1
