diff --git a/civilian.php b/civilian.php
index 8169d17..2eba4b6 100644
--- a/civilian.php
+++ b/civilian.php
@@ -713,6 +713,7 @@ This program comes with ABSOLUTELY NO WARRANTY; Use at your own risk.
                 <label class="col-lg-2 control-label">License Status</label>
                   <div class="col-lg-10">
                     <select name="civDL" class="form-control selectpicker civDL_picker" id="civDL" title="Select a license status" required>
+                <option value="Unobtained"> Unobtained </option>
                 <option value="Valid"> Valid </option>
                 <option value="Suspended"> Suspended </option>
                 <option value="Expired"> Expired </option>
-- 
2.17.1
