diff --git a/actions/civActions.php b/actions/civActions.php
index 61da118..d2dfadb 100644
--- a/actions/civActions.php
+++ b/actions/civActions.php
@@ -58,7 +58,7 @@ if (isset($_POST['create_warrant'])){
 function getCivilianNamesOwn()
 {
     $uid = $_SESSION['id'];
-	
+
     $link = mysqli_connect(DB_HOST, DB_USER, DB_PASSWORD, DB_NAME);
 
 	if (!$link) {
@@ -160,7 +160,7 @@ function ncicGetPlates()
     if (!$link) {
         die('Could not connect: ' .mysql_error());
     }
-    
+
 
     $query = 'SELECT ncic_plates.*, ncic_names.name FROM ncic_plates INNER JOIN ncic_names ON ncic_names.id=ncic_plates.name_id WHERE ncic_plates.user_id = "' . $uid . '"';
 
@@ -294,7 +294,7 @@ function create_name()
     $fullName = $_POST['civNameReq'];
     $firstName = explode(" ", $fullName) [0];
     $lastName = explode(" ", $fullName) [1];
-    
+
     //Set first name to all lowercase
     $firstName = strtolower($firstName);
     //Remove all special characters
@@ -308,7 +308,7 @@ function create_name()
     $lastName = preg_replace('/[^A-Za-z0-9\-]/', '', $lastName);
     //Set first letter to uppercase
     $lastName = ucfirst($lastName);
-	
+
 	$name = $firstName . ' ' . $lastName;
 
     $link = mysqli_connect(DB_HOST, DB_USER, DB_PASSWORD, DB_NAME);
@@ -368,7 +368,7 @@ function create_name()
 
     }
 
-    $_SESSION['identityMessage'] = '<div class="alert alert-success"><span>Successfully create an identity</span></div>';
+    $_SESSION['identityMessage'] = '<div class="alert alert-success"><span>Successfully created your identity!</span></div>';
 
     sleep(1);
     header("Location:".BASE_URL."/civilian.php#name_panel");
@@ -378,9 +378,9 @@ function create_name()
 function create_plate()
 {
 	session_start();
-	
+
 	$plate = $_POST['veh_plate'];
-	
+
     //Remove all spaces from plate
     $plate = str_replace(' ', '', $plate);
     //Set plate to all uppercase
@@ -389,11 +389,11 @@ function create_plate()
     $plate = str_replace('-', '', $plate);
     //Remove all special characters
     $plate = preg_replace('/[^A-Za-z0-9\-]/', '', $plate);
-	
+
     $vehicle = $_POST['veh_make_model'];
     $veh_make = explode(" ", $vehicle) [0];
     $veh_model = explode(" ", $vehicle) [1];
-	
+
     $uid = $_SESSION['id'];
 
     $submittedById = $_SESSION['id'];
@@ -445,7 +445,7 @@ function create911Call()
 	if (!$link) {
 		die('Could not connect: ' .mysql_error());
 	}
-	
+
 	$sql = "SELECT MAX(call_id) AS max FROM call_list";
 	$result=mysqli_query($link, $sql);
 
@@ -453,9 +453,9 @@ function create911Call()
 	{
 		$callid = $r['max'];
 	}
-	
+
 	$callid++;
-	
+
     $sql = "REPLACE INTO call_list (call_id) VALUES (?)";
 
 	try {
@@ -516,7 +516,7 @@ function edit_name()
     $fullName = $_POST['civNameReq'];
     $firstName = explode(" ", $fullName) [0];
     $lastName = explode(" ", $fullName) [1];
-    
+
     //Set first name to all lowercase
     $firstName = strtolower($firstName);
     //Remove all special characters
@@ -530,7 +530,7 @@ function edit_name()
     $lastName = preg_replace('/[^A-Za-z0-9\-]/', '', $lastName);
     //Set first letter to uppercase
     $lastName = ucfirst($lastName);
-	
+
 	$name = $firstName . ' ' . $lastName;
 
     $link = mysqli_connect(DB_HOST, DB_USER, DB_PASSWORD, DB_NAME);
@@ -589,7 +589,7 @@ function edit_name()
 
     }
 
-    $_SESSION['identityMessage'] = '<div class="alert alert-success"><span>Successfully Update an identity</span></div>';
+    $_SESSION['identityMessage'] = '<div class="alert alert-success"><span>Successfully updated the identity.</span></div>';
 
     sleep(1);
     header("Location:".BASE_URL."/civilian.php#name_panel");
@@ -598,9 +598,9 @@ function edit_name()
 function edit_plate()
 {
     session_start();
-    
+
     $plate = $_POST['veh_plate'];
-    
+
     //Remove all spaces from plate
     $plate = str_replace(' ', '', $plate);
     //Set plate to all uppercase
@@ -609,11 +609,11 @@ function edit_plate()
     $plate = str_replace('-', '', $plate);
     //Remove all special characters
     $plate = preg_replace('/[^A-Za-z0-9\-]/', '', $plate);
-    
+
     $vehicle = $_POST['veh_make_model'];
     $veh_make = explode(" ", $vehicle) [0];
     $veh_model = explode(" ", $vehicle) [1];
-    
+
     $uid = $_SESSION['id'];
 
     $submittedById = $_SESSION['id'];
@@ -635,7 +635,7 @@ function edit_plate()
         die('Could not connect: ' .mysql_error());
     }
 
-    
+
     $sql = "UPDATE ncic_plates SET name_id = ?, veh_plate = ?, veh_make = ?, veh_model = ?, veh_pcolor = ?, veh_scolor = ?, veh_insurance = ?, flags = ?, veh_reg_state = ?, notes = ? WHERE id = ?";
 
     try {
@@ -734,14 +734,14 @@ function create_warrant()
 function ncic_warrants()
 {
     $uid = $_SESSION['id'];
-	
+
    $link = mysqli_connect(DB_HOST, DB_USER, DB_PASSWORD, DB_NAME);
 
     if (!$link) {
         die('Could not connect: ' .mysql_error());
     }
 	$sql = 'SELECT ncic_names.id from ncic_names where submittedById = "' . $uid . '"';
-	
+
 	$results=mysqli_query($link, $sql);
 	while($row = mysqli_fetch_array($results, MYSQLI_BOTH))
         {
@@ -849,11 +849,11 @@ function delete_warrant()
 function create_weapon()
 {
 	session_start();
-	
+
     $weapon = $_POST['weapon_all'];
     $wea_type = explode(" ", $weapon) [0];
     $wea_name = explode(" ", $weapon) [1];
-	
+
     $uid = $_SESSION['id'];
 
     $submittedById = $_SESSION['id'];
@@ -978,4 +978,4 @@ function delete_weapon()
     $_SESSION['weaponMessage'] = '<div class="alert alert-success"><span>Successfully removed civilian weapon</span></div>';
     header("Location: ".BASE_URL."/civilian.php");
 }
-?>
\ No newline at end of file
+?>
-- 
2.17.1
