diff --git a/civilian.php b/civilian.php
index ba2d826..8169d17 100644
--- a/civilian.php
+++ b/civilian.php
@@ -559,7 +559,7 @@ This program comes with ABSOLUTELY NO WARRANTY; Use at your own risk.
                 <label class="col-lg-2 control-label">License Status</label>
                   <div class="col-lg-10">
                     <select name="civDL" class="form-control selectpicker" id="civDL" title="Select a license status" required>
-                <option value="Unobtained"> Valid </option>
+                <option value="Unobtained"> Unobtained </option>
                 <option value="Valid"> Valid </option>
                 <option value="Suspended"> Suspended </option>
                 <option value="Expired"> Expired </option>
--
2.17.1
