diff --git a/sql/oc_GTAV_data.sql b/sql/oc_GTAV_data.sql
index 7eaadc1..169f2e6 100644
--- a/sql/oc_GTAV_data.sql
+++ b/sql/oc_GTAV_data.sql
@@ -1,7 +1,3 @@
---
--- Database: `opencad_dev`
---
-
 --
 -- Dumping data for table `colors`
 --
@@ -260,15 +256,15 @@ INSERT INTO `colors` (`id`, `color_group`, `color_name`) VALUES
 -- Dumping data for table `departments`
 --
 
-INSERT INTO `departments` (`department_id`, `department_name`) VALUES
-(1, 'Communications'),
-(2, 'State'),
-(3, 'Highway'),
-(4, 'Sheriff'),
-(5, 'Police'),
-(6, 'Fire'),
-(7, 'EMS'),
-(8, 'Civilian');
+INSERT INTO `departments` (`department_id`, `department_name`, `department_short_name`, `department_long_name`) VALUES
+(1, 'Communications', 'SAECOMM', 'San Andreas Emergency Communications', 0),
+(2, 'State', 'SASP', 'San Andreas State Police', 0),
+(3, 'Highway', 'SAHP', 'San Andreas Highway Patrol', 0),
+(4, 'Sheriff', 'BCSO', 'Blaine County Sheriffs\' Office', 0),
+(5, 'Police', 'SAPD', 'San Andreas Police Department', 0),
+(6, 'Fire', 'SAFD', 'San Andreas Fire Department', 0),
+(7, 'EMS', 'SAEMS', 'San Andreas Emergency Services', 0),
+(8, 'Civilian', 'SACS', 'San Andreas Civilian Services', 0);
 
 --
 -- Dumping data for table `genders`
diff --git a/sql/oc_structure.sql b/sql/oc_structure.sql
index 1ebfc33..200a06b 100644
--- a/sql/oc_structure.sql
+++ b/sql/oc_structure.sql
@@ -136,9 +136,10 @@ CREATE TABLE `colors` (
 
 CREATE TABLE `departments` (
   `department_id` int(11) NOT NULL,
-  `department_name` varchar(255) DEFAULT NULL COMMENT 'The functional name of the department. (eg. Police, Fire, EMS)',
-  `department_long_name` varchar(255) DEFAULT NULL COMMENT 'The name of the department. (eg. Los Angeles Police Department, Blaine County Sheriffs\' Office',
-  `department_short_name` varchar(6) DEFAULT NULL COMMENT 'The acronym of the department name. (eg. BCSO, LAPD, LAFD)'
+  `department_name` varchar(255) DEFAULT NULL,
+  `department_short_name` varchar(10) NOT NULL,
+  `department_long_name` varchar(255) NOT NULL,
+  `allow_department` int(11) NOT NULL
 ) ENGINE=InnoDB DEFAULT CHARSET=latin1 ROW_FORMAT=COMPACT;
 
 -- --------------------------------------------------------
-- 
2.18.0
