diff --git a/actions/profileActions.php b/actions/profileActions.php
index 09edc78..7619602 100644
--- a/actions/profileActions.php
+++ b/actions/profileActions.php
@@ -99,22 +99,32 @@ function getMyRank()
 
 function changePassword()
 {
-  $con = mysqli_connect(DB_HOST, DB_USER, DB_PASSWORD, DB_NAME);
-  $result = mysqli_query($con, "SELECT * FROM `users`") or die(mysqli_error($con));
-  $row = mysqli_fetch_array($result);
+
+  error_reporting(E_ALL);
+  ini_set('display_errors', 1);
+
+  // Connect to database
+  $link = mysqli_connect(DB_HOST, DB_USER, DB_PASSWORD, DB_NAME);
+
+  if (!$link) {
+    die('Could not connect: ' .mysql_error());
+  }
+
+  // Get users
+  $query = mysqli_query($link, "SELECT * FROM `users`") or die(mysqli_error($link));
+
+  $row = mysqli_fetch_array($query);
   $id = $row['id'];
   $password = $row['password'];
-  $id = $_SESSION['id'];
-  if (isset($_POST['changePassword'])) {
   $newpassword = $_POST['password'];
   $hashed_password = password_hash($newpassword, PASSWORD_DEFAULT);
-  mysqli_query($con,"UPDATE `users` SET `password` = '$hashed_password' WHERE `id` = '$id'") or die(mysqli_error($con));
+  mysqli_query($link,"UPDATE `users` SET `password` = '$hashed_password' WHERE `id` = '$id'") or die(mysqli_error($link));
 
-  $_SESSION['passwordUpdate'] = '<div class="alert alert-success"><span>Your password has been updated!</span></div>';
+  $_SESSION['changePassword'] = '<div class="alert alert-success"><span>Password successfully updated.</span></div>';
 
   sleep(1); //Seconds to wait
-  header("Location: ".BASE_URL."/profile.php");
-}
+  echo $_SESSION['changePassword'];
+  header("Location: ".BASE_URL."/profile.php?changePassword=true");
 }
 
 function getRanks()
diff --git a/profile.php b/profile.php
index fe8fd14..a59e7a5 100644
--- a/profile.php
+++ b/profile.php
@@ -15,6 +15,7 @@ This program comes with ABSOLUTELY NO WARRANTY; Use at your own risk.
 require("./oc-config.php");
 
 include("./actions/profileActions.php");
+
     session_start();
 
     // TODO: Verify user has permission to be on this page
@@ -36,7 +37,13 @@ include("./actions/profileActions.php");
         unset($_SESSION['profileUpdate']);
     }
 
-var_dump($_SESSION);
+
+    if (isset($_GET['changePassword']))
+    {
+        $changePassword = '<div class="alert alert-success"><span>Password successfully updated.</span></div>';
+        unset($_SESSION['changePassword']);
+    }
+
 ?>
 
 <!DOCTYPE html>
@@ -119,6 +126,7 @@ var_dump($_SESSION);
                     <span class="fas fa-angle-down"></span>
                   </a>
                   <ul class="dropdown-menu dropdown-usermenu pull-right">
+                    <li><a href="<?php echo BASE_URL; ?>/profile.php"><i class="fas fa-user pull-right"></i> My Profile</a></li>
                     <li><a href="<?php echo BASE_URL; ?>/actions/logout.php"><i class="fas fa-sign-out-alt pull-right"></i> Log Out</a></li>
                   </ul>
                 </li>
@@ -156,9 +164,8 @@ var_dump($_SESSION);
                     <div class="clearfix"></div>
                   </div>
                   <!-- ./ x_title -->
-                  <?php echo $passwordUpdate;?>
                   <div class="x_content">
-                  <?php echo $profileUpdate;?>
+                  <?php echo $profileUpdate, $changePassword;?>
                   <form action="<?php echo BASE_URL; ?>/actions/profileActions.php" method="post" class="form-horizontal">
                   <fieldset>
                     <div class="form-group">
-- 
2.18.0
